### DISCUSSION TITLE
The node-resolve plugin seems to break Vue when the Vue app imports a lib and consumes a native web components.

### DISCUSSION THREAD

**Discussion by jrgleason** [rollup/plugins#1684] [Q&A]
- Rollup Plugin Name: node-resolve
- Rollup Plugin Version: 15.2.3
- Rollup Version: 2.79.1
- Operating System (or Browser): OSX
- Node Version: 22
- Link to reproduction _(⚠️ read below)_: https://github.com/vuejs/core/issues/10365

### Expected Behavior

I would like to generate component libraries using rollup and MUI. To do this I need to use the node-resolve plugin. However, when I try to import the components created by rollup and the plugin I get an error (see the Vue issue). If I remove the plugin everything seems to work but that requires me also removing MUI. 

### Actual Behavior

I would like to be able to use web components generated using rollup and node-resolve in a vue project

### Additional Information

I think the easiest thing to do here is reference the Vue issue. There is a link to the github repo for the example project and an explination in there. Let me know if you want me to add or duplicate additional information to this post.

**Comment by jrgleason** [comment]
So was this moved to a discussion because the onus is on Vue for sure? Are they not handling something in a standard way?

**Reply by shellscape** [reply]
We're moving issues that don't meet the requirements laid out in the template, instead of closing them. Reproductions from other issues in other repos don't meet the criteria for a reproduction that's very clearly explained in the issue template.
